### PR TITLE
clarification for release steps

### PR DESCRIPTION
**Pull request by macfarla**
### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [ ] unit tests: `./gradlew build`
- [ ] acceptance tests: `./gradlew acceptanceTest`
- [ ] integration tests: `./gradlew integrationTest`
- [ ] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -11,10 +11,14 @@ assignees: ''
...
   - [ ] Notify maintainers about updating changelog for in-flight PRs 
 - [ ] Optional: for hotfixes, create a release branch and cherry-pick, e.g. `release-<version>-hotfix`
-- [ ] Optional: create a PR into main from the hotfix branch to see the CI checks pass
+  - [ ] Optional: for hotfixes, create a PR into main from the hotfix branch to see the CI checks pass
 - [ ] On the appropriate branch/commit, create a calver tag for the release candidate, format example: `24.4.0-RC2`
-- [ ] Sign-off with team; confirm tag is correct in #besu-release in Discord
+  - [ ] git tag 24.7.1-RC1
+  - [ ] git push upstream 24.7.1-RC1
+- [ ] Sign-off with team; announce the tag in #besu-release in Discord
+  - [ ] Targeting this tag for the burn-in: 24.7.1-RC1
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:19]:
```suggestion
  - [ ] Targeting this tag for the burn-in: https://github.com/hyperledger/besu/releases/tag/24.7.1-RC1
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -11,10 +11,14 @@ assignees: ''
...
 - [ ] Consensys staff start burn-in using the proposed release <version-RCX> tag
+- [ ] ... Wait for 3 days
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:21]: The number of days could be variable. Not sure we need this line at all tbh...seems implied by "sign off burn-in" to me?

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -11,10 +11,13 @@ assignees: ''
 - [ ] Update changelog if necessary, and merge a PR for it to main
```
Comment (jframe) [inline .github/ISSUE_TEMPLATE/release-checklist.md:11]: I think this was moved to later in the process. Would moving it before this step `Using the same git sha, create a calver tag for the FULL RELEASE, example format `24.4.0` make sense?

**Comment by siladu** [comment]
Wonder if we should add an item to ensure someone is monitoring social channels to keep an eye out for any release issues users are having.

**Comment by non-fungible-nelson** [comment]
is there anywhere we are capturing consensys-specific release tasks? like notify our internal folks of breaking changes / default swaps, more?

**Comment by siladu** [comment]
> is there anywhere we are capturing consensys-specific release tasks? like notify our internal folks of breaking changes / default swaps, more?

Think that would currently fall under "Social announcements".
We could expand on that to add a list of channels.

**Review by siladu — APPROVED** [review]

**Pull request merged**